lib/oppen/print_stack.rb

```diff
@@ -73,16 +74,18 @@ def output
     #
     # @param token [Token]
     # @param token_width [Integer]
+    # @param trim_on_break [Integer|Nil] Number of chars to trim in case of new line in break.
+    #                                    If nil, never trims.
```

Comment:
```suggestion
    # @param trim_on_break [Integer] Number of trailing whitespace characters to trim.
```

I am disliking the fact it's nillable. It should be 0 by default.